src/tools/sign-oci-layer-root-hashes/src/registry.rs

```diff
@@ -406,19 +407,75 @@ fn decompress_file(compressed_path: &Path, decompressed_path: &Path) -> Result<(
     Ok(())
 }
 
-fn attach_tarfs_index(path: &Path) -> Result<()> {
-    info!("Adding tarfs index to layer");
-    let mut file = std::fs::OpenOptions::new()
-        .read(true)
+fn attach_erofs_meta(path: &Path) -> Result<()> {
+    info!("Creating erofs meta. Appending decompressed tar to erofs meta");
+
+    // Create an erofs image using mkfs.erofs
+    let erofs_path = path.with_extension("erofs");
+    debug!(
+        "Creating erofs meta image {:?} from {:?}",
+        &erofs_path, &path
+    );
+
+    let status = Command::new("mkfs.erofs")
+        .args([
+            "--tar=i",
+            "-T", "0", // zero out unix time
+            "-U", "c1b9d5a2-f162-11cf-9ece-0020afc76f16", // set UUID to something specific
```

Comment:
Would be good as a constant between solar and TS.

Reply:
Created an erofs-common package that has the alignment and uuid as constants.